src/LaravelRequestDocsToOpenApi.php

```diff
@@ -33,9 +33,18 @@ public function openApi(array $docs): LaravelRequestDocsToOpenApi
     private function docsToOpenApi(array $docs): void
     {
         $this->openApi['paths'] = [];
+        $includeBodyForDelete   = config('request-docs.open_api.include_body_on_delete', false);
+        $excludeHttpMethods     = array_map(fn ($item) => strtolower($item), config('request-docs.open_api.exclude_http_methods', []));
+
         foreach ($docs as $doc) {
-            $requestHasFile  = false;
```

Comment:
This declaration is necessary 

https://github.com/rakutentech/laravel-request-docs/blob/64325d680b1d2bd738300f9e61073b869ca6b4b3/src/LaravelRequestDocsToOpenApi.php#L64-L75

Reply:
Sorry it has been moved down.